types/connect-busboy/index.d.ts

```diff
@@ -15,4 +16,12 @@ declare namespace connectBusboy {
     }
 }
 
+declare global {
+    namespace Express {
+        export interface Request {
```

Comment:
If you just remove the `export` here you should be able to remove the lint rule exceptions